.github/workflows/ci.yml

```diff
@@ -9,22 +9,22 @@ on:
 jobs:
   test:
     name: Elixir ${{matrix.pair.elixir}} / OTP ${{matrix.pair.otp}}
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     env:
       MIX_ENV: test
     strategy:
       fail-fast: false
       matrix:
         include:
           - pair:
-              elixir: 1.7.4
-              otp: 19.3.6.13
```

Comment:
I got this CI error before I made this change:

>Run erlef/setup-elixir@v1
Error: Requested Erlang/OTP version (19.3.6.13) not found in version list (should you be using option 'version-type': 'strict'?)